scripts/httpd/upload.cgi

```diff
@@ -160,6 +160,41 @@ def emit_fedmsg(config, name, checksum, filename, username, msgpath):
         except Exception as e:
             sys.stderr.write("Error with fedmsg", str(e))
 
+    # Emit a fedmsg-messaging message
+    if config.getboolean('upload', 'fedora-messaging', fallback=True):
+        try:
+            import fedora_messaging.api
+            import fedora_messaging.config
+            import fedora_messaging.exceptions
+
+            if config['upload'].get('fedora-messaging-config'):
+                fedora_messaging.config.conf.load_config(
+                    config['upload']['fedora-messaging-config'])
+
+            message = dict(
+                name=name,
+                md5sum=checksum,
+                filename=filename.split('/')[-1],
+                agent=username,
+                path=msgpath
+            )
+
+            msg = fedora_messaging.api.Message(
+                topic="lookaside.new",
+                body=message
+            )
+            fedora_messaging.api.publish(msg)
+        except ImportError:
+            _log.warning("Fedora messaging does not appear to be available")
```

Comment:
I don't see a `_log` variable in this file.

Reply:
oups, bad copy/pasta, thanks for catching it